jenkins-design-language/src/js/components/material-ui/SvgIcon/SvgIcon.js

```diff
@@ -70,24 +56,18 @@ class SvgIcon extends Component {
       ...other
     } = this.props;
 
-    const offColor = color ? color : 'white';
-    const onColor = hoverColor ? hoverColor : offColor;
-
     const mergedStyles = Object.assign({
-      display: 'inline-block',
-      fill: this.state.hovered ? onColor : offColor,
       height: size ? size : 24,
       width: size ? size : 24,
       verticalAlign: 'middle',
-      userSelect: 'none',
-      transition: 'all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms', 
     }, style);
 
     delete other.iconName;
 
     return (
       <svg
         {...other}
+        className="material-icon"
```

Comment:
Not sure if there are any instances, but `className` might get swallowed if it's passed in `other`